packages/app-graphql/README.md

```diff
@@ -39,7 +39,7 @@ module.exports = {
 | Option         | Type     | Default                               | Description                                                                                          |
 | -------------- | -------- | ------------------------------------- | ---------------------------------------------------------------------------------------------------- |
 | `cors`         | `Object` | `{ origin: true, credentials: true }` | Options passed directly to the express [cors](https://github.com/expressjs/cors) middleware package. |
-|  |
+|                |          |                                       |                                                                                                      |
```

Comment:
Delete the row? This leaves an empty row.

Reply:
*shrug* I'm just doing what prettier tells me to.